app/Http/Controllers/Api/CategoryController.php

```diff
@@ -24,6 +24,16 @@ public function index()
         return response()->json($category);
     }
 
+    /**
+     * Show all the categories without paginate function
+    */
+    public function showAll()
+    {
+        $category = Category::all();
```

Comment:
change variable name to `$categories`, since you're getting `all` and not only one. This does not affect functionality but it improves readability of your code.

Reply:
This is resolved ma'am.